src/dsp_tools/commands/excel2json/project.py

```diff
@@ -23,11 +23,12 @@ def excel2json(
 
         data_model_files
         |-- lists
-        |   |-- de.xlsx
-        |   `-- en.xlsx
+        |   |-- list.xlsx
+        |   `-- list_2.xlsx
         `-- onto_name (onto_label)
             |-- properties.xlsx
             `-- resources.xlsx
+        | -- json_header.xlsx (optional)
```

Comment:
This is not a correct tree

Reply:
then I don't know how it should look like.